packages/coingecko/package.json

```diff
@@ -38,8 +38,10 @@
     "@ethersproject/contracts": "^5.6.0",
     "@ethersproject/providers": "^5.6.2",
     "@usedapp/core": "workspace:*",
-    "isomorphic-fetch": "^3.0.0",
-    "react": "^17.0.1"
+    "isomorphic-fetch": "^3.0.0"
```

Comment:
```suggestion
    "isomorphic-fetch": "^3.0.0",
    "react": "^17.0.1"
```

Reply:
Add react to depedencies and pnpm install again.

Reply:
I have app using react 18 and your suggestion will cause Invalid Hook Call Error

Reply:
added to devDependencies